build/package/agent/Dockerfile_cross

```diff
@@ -1,23 +1,22 @@
-FROM minio/mc as minio_client
-
-FROM mysql
-
-COPY --from=minio_client /usr/bin/mc /usr/bin/mc
-
-RUN mkdir -p /opt/vxbinaries && \
-    microdnf install jq && \
-    microdnf clean all
-
-# Entrypoint script
-COPY build/package/agent/_tmp/startup.sh /opt/vxbinaries/
-COPY build/package/agent/_tmp/version /opt/vxbinaries/
-
-COPY build/package/agent/_tmp/windows /opt/vxbinaries/windows
-COPY build/package/agent/_tmp/linux   /opt/vxbinaries/linux
-COPY build/package/agent/_tmp/darwin  /opt/vxbinaries/darwin
-
-WORKDIR /opt/vxbinaries
-
-RUN chmod +x /opt/vxbinaries/startup.sh
-
-ENTRYPOINT ["/opt/vxbinaries/startup.sh"]
+FROM minio/mc as minio_client
+
+FROM alpine
```

Comment:
Don't think this is good idea to use alpine here since we build everything in debian

Reply:
This image was actually built on oraclelinux:8-slim (base for mysql), not debian. Also, its function is only to load agent distributions into minio and create some entries to the database. For optimize its size, I consider it possible to use alpine